### PR TITLE
Add link to SIG list from first contribution page

### PR DESCRIPTION
**Pull request by cainejette**
Looks like https://github.com/kubernetes/community/pull/5182/files broke the link

### CODE REVIEW

File: contributors/guide/first-contribution.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ When you've found an issue to work on, you can assign it to yourself.
 ## Learn about SIGs
 
 Some repositories in the Kubernetes Organization are owned by 
-[Special Interest Groups], or SIGs.
+[Special Interest Groups](./sig-list.md), or SIGs.
```
Comment (mrbobbytables) [inline contributors/guide/first-contribution.md:60]:
```suggestion
[Special Interest Groups][sl], or SIGs.
```
`./sig-list.md` would map to a file in the same directory and still break.

The link is there, but at the bottom of the doc as a reference link (`[sl]`)

```
[sl]: /sig-list.md
```

**Review by mrbobbytables — CHANGES_REQUESTED** [review]

**Comment by mrbobbytables** [comment]
Thanks! - one last thing - before we can accept the PR, can you sign the CLA?^^;;

**Comment by cainejette** [comment]
@mrbobbytables I don't plan to sign the CLA so please feel free to take this PR over

**Comment by mrbobbytables** [comment]
Thanks for letting us know!
/close

**Pull request closed without merging**